website/sidebars.js

```diff
@@ -774,6 +774,24 @@ const sidebarSettings = {
             "guides/orchestration/custom-cicd-pipelines/5-something-to-consider",
           ],
         },
+        {
+          type: "category",
+          label: "Use dbt Cloud's webhooks with other SaaS apps",
```

Comment:
suggesting shortening this sidebar title

```suggestion
          label: "Webooks with dbt Cloud and SaaS apps",
```